### DISCUSSION TITLE
Include note location in YAML

### DISCUSSION THREAD

**Discussion by jdlee** [SilentVoid13/Templater#209] [Ideas & Feature Requests]
Is it possible to automatically include the location of the note creation in the YAML.  This would mimic Evernote’s automatic capture of location in the note.

**Comment by ChristinWhite** [comment]
Are you talking about your physical location or the note path?

If you want a path you can that information from the File module. For instance: 

```javascript
absolutePath = tp.file.path()
```

The relative path from your vault's root looks like it's a bit more work but there are several approaches you could take.

I'm not sure how you'd get physical location properties but I'm guessing it's possible, but may not be easy.
